lib/improver/utilities/spatial.py

```diff
@@ -49,54 +49,77 @@
 MAX_DISTANCE_IN_GRID_CELLS = 500
 
 
-def check_if_grid_is_equal_area(cube):
+def check_if_grid_is_equal_area(cube, require_equal_xy_spacing=True):
     """Identify whether the grid is an equal area grid.
     If not, raise an error.
```

Comment:
As this function isn't really testing for equal-area grids, you should include in the doc-string what it actually does.